### ISSUE TITLE
Can't unlock motocycles

### ISSUE THREAD

**Issue by KingRyuShin** [Xseba360/esx_locksystem#4]
There is a problem with the script..
We can take the key and lock the motocycles but once you try to unlock it, it say that the plate is not found that problem only happen with motocycles only.

EDIT: Also sometime, people can hear the lock and unlock sound across the map

**Comment by Xseba360** [comment]
I'll look into the motorcycles, but the second bug you mentioned is caused by solo session as far as i know.

**Comment by KingRyuShin** [comment]
Ok, I will try to check if the player is in a Solo Session when it happen

**Comment by KingRyuShin** [comment]
yeah, for the sound bug it's really when a player is in a solo session..
Also any news for the motocycle not having plate?

**Comment by ItsJokaful** [comment]
When i sit on a motorcycle i can lock & unlock it.
But when i stand next to it i cant lock or unlock it.
